lib/rules/require-valid-css-selector-in-test-helpers.js

```diff
@@ -0,0 +1,158 @@
+'use strict';
+
+const { getPropertyValue, parseCallee } = require('../utils/utils');
+const cssTree = require('css-tree');
+
+const TEST_HELPER_IMPORTS = new Set(['fillIn', 'findAll', 'find', 'click', 'focus']);
```

Comment:
This is missing a few helpers (full list in https://github.com/emberjs/ember-test-helpers/blob/master/API.md):

Needs to add:

* `blur`
* `doubleClick`
* `scrollTo`
* `select`
* `tap`
* `triggerEvent`
* `triggerKeyEvent`
* `typeIn`
* `waitFor`

(I didn't make a suggestion because I'm certain that I would have gone awry of prettier preferences 😝 )

Reply:
Fixed